### ISSUE TITLE
Consider using Yambar as bar and Fuzzel as launcher.

### ISSUE THREAD

**Issue by uncomfyhalomacro** [openSUSE/openSUSEway#72]
## Choosing Yambar
Reasons: Minimalist setup
Cons: Yambar does not have tray support

## Choosing Fuzzel
Reasons: Very minimal dependencies. Looks good and easy to setup. Wofi uses GTK which is somewhat unnecessary for a launcher. Also wofi is currently unmaintained. Though there are bug fix and documentation related commits, it seems the author is not removing the "not actively maintained" header soon on the sourcehut website.
Cons: Not GTK. No submenus.

Example setup (not swaywm but riverwm):
![image](https://user-images.githubusercontent.com/66054069/178220394-6e635ce5-e11c-4023-94fc-f4abc5693f40.png)

**Comment by jubalh** [comment]
> No GTK. Very few dependencies.

Those reasons will only make sense if we not already depend on these things already anyways.

**Comment by mcepl** [comment]
> > No GTK. Very few dependencies.
> 
> Those reasons will only make sense if we not already depend on these things already anyways.

Yes, but if we want be pushing in that direction we will never get there.

**Comment by uncomfyhalomacro** [comment]
Closing issue. Tbf, waybar is good for apps that have a systray feature. So I don't think this will be moved forward in the future as yambar does not have a systray support yet.

**Comment by mcepl** [comment]
On Thu Feb 1, 2024 at 1:27 PM CET, Krish Jain wrote:
> PS: Although Yambar is a good option, we are currently using Waybar, which is well-maintained. So, let’s continue using Waybar for now

I won’t argue about waybar (I personally use swaybar and it is
doing just fine), but I would say a word on the replacement
of wofi. Yes, it should be replaced, because the upstream is
dead, but I would argue for rofi (particularly its Wayland
port, `rofi-wayland` in openSUSE). Its main advantage is that
it has much wider area of use than just a dmenu replacement
(https://github.com/davatorium/rofi/wiki/User-scripts),
I have written couple of scripts using it
(https://git.cepl.eu/cgit/nmcli-rofi/,
https://git.cepl.eu/cgit/pinentry-rofi/) and it works just fine
for that purpose.

-- 
http://matej.ceplovi.cz/blog/, @***@***.***
GPG Finger: 3C76 A027 CA45 AD70 98B5  BC1D 7920 5802 880B C9D8
 
I am a Roman Catholic, so that I do not expect “history” to be
anything but a “long defeat” -- though it contains (and in
a legend may contain more clearly and movingly) some samples or
glimpses of final victory.
      -- J.R.R. Tolkien

**Comment by denisok** [comment]
I don't mind rofi personally. For my use-case - I need something to launch apps and that it. Maybe discussions and votes: https://github.com/openSUSE/openSUSEway/discussions ?

BTW wofi just got updated: https://hg.sr.ht/~scoopta/wofi

@krishjainx if you would like to change something - you can always start conversation and see if there is an interest. Or maybe propose a patch with demonstration how it works.

**Comment by FilippoBonazziSUSE** [comment]
Yes, looks like the wofi banner has changed as well:

> This project is not being actively maintained. It does still receive some updates from time to time but nothing frequent.

This makes it sound like it's at least not fully abandoned. So for me there is no pressing need to immediately replace it with an alternative just for this reason.

I am still open to replace it with an alternative if an alternative is better. I might look into rofi if people have been using that and are happy with it.

**Comment by FilippoBonazziSUSE** [comment]
I have opened https://github.com/openSUSE/openSUSEway/discussions/140 as a single place to discuss wofi replacement, so that we don't lose information in old closed issues like this one.
